addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/handler/YamahaBridgeHandler.java

```diff
@@ -232,35 +261,45 @@ public void initialize() {
         }
 
         String host = (String) thing.getConfiguration().get(YamahaReceiverBindingConstants.CONFIG_HOST_NAME);
+        BigDecimal port = (BigDecimal) thing.getConfiguration().get(YamahaReceiverBindingConstants.CONFIG_HOST_PORT);
 
-        if (host == null || host.isEmpty()) {
+        if (host == null || port == null || host.isEmpty()) {
```

Comment:
The `port `is a little bit strange in the middle and maybe you could use StringUtils.isEmpty it combines the two checks for the `host`.